ajax/libs/canvas-nest.js/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "canvas-nest.js",
+  "filename": "canvas-nest.min.js",
+  "version": "1.0.0",
+  "description": "A nest backgroud of website draw on canvas use javascript, do not depend on jQuery. Less then 2kb.",
+  "homepage": "http://www.atool.org/",
+  "keywords": [
+    "canvas",
+    "html5",
+    "nest"
+  ],
+  "autoupdate": {
```

Comment:
There is no valid git tag in its repository so auto-update depends on git will not work.